### ISSUE TITLE
Obtained glTF Model isn’t rendered properly

### ISSUE THREAD

**Issue by Pratyush1184** [facebookincubator/FBX2glTF#185]
I am using a Ubuntu 18.04.2 LTS(64-bit) on my machine with 16 GB of RAM.
I compiled the code with cmake using the command given in the README file:
cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Release
On getting a dependency error due to libxml2, I installed the package libxml2-dev using the command:
sudo apt-get install libxml2-dev
The compilation was successful and then I run the following command:
make -Cbuild -j4 install
I obtained an executable file named FBX2glTF-linux-x64.
When I used this executable file to test with a sample FBX model that I uploaded(HP_Model), I obtained the corresponding GLTF files:
On viewing the GLTF models on Babylon sandbox(https://sandbox.babylonjs.com/), the models appeared partial and incomplete with some missing entities. I am attaching the screenshots for the rendered model and also the corresponding FBX models with their rendered view on AutoDesk viewer.
![Autodesk_Model_View](https://user-images.githubusercontent.com/20305094/57060349-bb23d680-6cd6-11e9-97e0-62e405ec674e.png)
![Sandbox_Model_View_1](https://user-images.githubusercontent.com/20305094/57060350-bb23d680-6cd6-11e9-8c82-4ed70ef28558.png)
![Sandbox_Model_View_2](https://user-images.githubusercontent.com/20305094/57060351-bbbc6d00-6cd6-11e9-98d2-306ca2fc517f.png)
[HP_Model.zip](https://github.com/facebookincubator/FBX2glTF/files/3136979/HP_Model.zip)

**Comment by johnmarston2** [comment]
Facing similar issue, Any help related to fixing it is much appreciated.

**Comment by avpn989** [comment]
Yes, I also encountered similar issues. The rendered model in my case is much worse.

**Comment by Pratyush1184** [comment]
Exactly, I even tried the pre-compiled binaries(https://github.com/facebookincubator/FBX2glTF/releases?fbclid=IwAR0uyANQOxrPeAKRbxiGG4-mYhPB7FoNTXkfpZccaFDcKlP9-STMP5MVYLQ) and the results are the same.

**Comment by zellski** [comment]
Hi. You already posted this exact issue months ago: https://github.com/facebookincubator/FBX2glTF/issues/164

**Comment by Pratyush1184** [comment]
The issue I raised months ago was due to errors in the compilation process. This issue is about the pre-compiled builds that I used and obtained the same results. Atleast have a look into the conversion at your side and check the rendered model before closing this issue.

**Comment by zellski** [comment]
I didn’t close it because it’s not an issue, I closed it because there is no action to take. If you re-read my response there, the issue is that the FBX SDK (Autodesk’s code) simply doesn’t handle the broken geometry in your asset correctly. The fact that their online viewer *does* parse the geometry correctly is nice for you, but meaningless to the issue at hand.

In an ideal world there would be a fix for this. In this world there is not a reasonable fix that comes to mind.

**Comment by zellski** [comment]
@johnmarston2 @avpn989 please open separate issues, and if you can include your FBX assets that’ll help.

**Comment by Pratyush1184** [comment]
I tried to file an issue with the FBX SDK but could not find a repository? Could you please redirect me to their repository where I could file this issue...

**Comment by zellski** [comment]
@Pratyush1184 FBX is a commercial product. There is no GitHub repository for the SDK. You can try to file a request in Autodesk's forums, but I wouldn't hold my breath -- they don't really support the software in any meaningful way anymore: https://forums.autodesk.com/t5/fbx-forum/bd-p/area-b64
